DragonKitchen/DragonKitchen/Views/ColorQuestView.swift

```diff
@@ -57,10 +53,10 @@ struct ColorButton : View {
                 GridItem(.fixed(100))
             ],
             spacing: 13) {
-                ForEach(colors.indices) { index in
+                ForEach(colors.indices, id: \.self) { index in
```

Comment:
사용하지 않는 변수는 "_"로 작성해보는건 어떨까요?

```suggestion
                ForEach(colors.indices, id: \.self) { _ in
```

Reply:
오호 index가 안쓰는 변수였군요! 짚어주셔서 감사합니다 👍🏻